src/typography/_fonts_config.scss

```diff
@@ -21,6 +25,16 @@ $serif-stack: Georgia, serif;
  }
 }
 
+%calibre-medium {
+ font-family: $sans-stack;
+ font-weight: 400;
+ font-style: normal;
```

Comment:
@misscs I don't know tbh, we have been doing it as a convention for all of the font extends... seems likely that it's unneccessary